### DISCUSSION TITLE
更改Windows终端的字体后，中文显示乱码的问题

### DISCUSSION THREAD

**Discussion by moli721** [RipplePiam/MobaXterm-Chinese-Simplified#66] [Q&A]
很久一段时间我使用MobaXterm都能正常显示中文，但是最近一个月没用，突然登录进去发现中文乱码，如图
![2024-8-30](https://github.com/user-attachments/assets/9c6b5e61-2eec-42ad-80de-6a81d77115e0)
只有中文会乱码，但是英文不会，不知道是不是我近一个月中将命令行工具中的字体改成Hasklug Nerd Font的缘故

**Comment by RipplePiam** [comment]
应该和汉化没关系。
仅拷贝mobaxterm.exe和CygUtils.plugin到一个干净的文件夹中，重新打开mobaxterm应该没问题

**Comment by moli721** [comment]
重新拷贝到新的文件夹仍不能解决，如图
![1725022755248](https://github.com/user-attachments/assets/c67a1eb0-825a-4c77-99f1-2f3a876203ad)

![1725022767840](https://github.com/user-attachments/assets/bed7cc0b-02ed-4245-af21-af3ded43cab1)

我甚至下载了你最新发布的MobaXterm_Portable_v24.2版本，仍然出现相同的问题，不知道是哪里的问题，

系统配置：win11
命令行使用的字体：Hasklug Nerd Font

**Comment by RipplePiam** [comment]
字体问题，和汉化无关。
个性化的设置请自行摸索，可尝试设置MXT终端字符集、编码等，但涉及远程终端兼容问题，不建议使用，自行斟酌。
与汉化无关issue请自行关闭，或转至讨论板块寻求帮助。
